app/views/management/schools/show.html.erb

```diff
@@ -10,6 +10,8 @@
     <% if @school.configuration %>
       <%= link_to "Review your school's energy analysis", school_analysis_index_path(@school), class: 'btn btn-outline-dark rounded-pill font-weight-bold' %>
     <% end %>
+    <%= link_to "Download your data", school_downloads_path(@school), class: 'btn btn-outline-dark rounded-pill font-weight-bold' if can?(:read, :school_downloads) %>
```

Comment:
Thought ewe said this was going under the My School menu?

Reply:
It is, but you don't get the My School menu if you're a management type - it's on the menu too.